portfolio/src/main/java/com/google/sps/data/Task.java

```diff
@@ -53,7 +81,28 @@ public Duration getDuration() {
     return duration;
   }
 
-  public Integer getPriority() {
+  public TaskPriority getPriority() {
+    return priority;
+  }
+
+  public Integer getPriorityInt() {
```

Comment:
I don't think this method is necessary. I think the callers can call getPriority() and then call getPriority() on that.

Reply:
Done.